app/javascript/src/Common/components/SelectWithModal.tsx

```diff
@@ -52,11 +52,12 @@ const SelectWithModal = <T extends IRecord>({
   helperTextInvalid,
   fetchItems
 }: Props<T>): React.ReactElement => {
+  let { current: isOnMount } = useRef(true)
```

Comment:
Interesting. I guess this is an alternative to the standard:
```
let isOnMount = useRef(true)

if (isOnMount.current) {
  isOnMount.current = false
}
```

Reply:
I don't understand this. May somebody explain?

Reply:
This syntax:
```js
let { current: isOnMount } = useRef(true)
```
Is called destructuring and it is an equivalent for:
```js
const ref = useRef(true)
let isOnMount = ref.current
```

Reply:
Honestly, the second one is clearer :grimacing:

Reply:
I guess destructuring 1 prop does look unnecessary verbose but it's the way modern JS is written 🤷🏻.